docs/knowledge_base/tutorials/chatbot/index.md

```diff
@@ -0,0 +1,289 @@
+# Creating a Chatbot Website using Taipy
+
+In this tutorial we will create a simple chatbot website using Taipy.
+
+[Try it live](https://demo-llm-chat.taipy.cloud/){: .tp-btn target='blank' }
+[Get it on GitHub](https://github.com/Avaiga/demo-llm-chat){: .tp-btn .tp-btn--accent target='blank' }
+
+<p align="center">
+  <img src="chatbot_roundconv.png" alt="Render of the app" width="100%"/>
+</p>
+
+
+Here we will use OpenAI's API with GPT-3. This tutorial can easily
+be adapted to other LLMs.
+
+# Table of Contents
+- [Step 1: Install Requirements](#step-1-install-requirements)
+- [Step 2: Imports](#step-2-imports)
+- [Step 3: Initialize variables](#step-3-initialize-variables)
+- [Step 4: Create a function to generate responses](#step-4-create-a-function-to-generate-responses)
+- [Step 5: Create a function to add the new messages to the conversation](#step-5-create-a-function-to-add-the-new-messages-to-the-conversation)
+- [Step 6: Create the User Interface](#step-6-create-the-user-interface)
+- [Step 7: Run the application](#step-7-run-the-application)
+- [Step 8: Styling](#step-8-styling)
+- [Step 9: More features](#step-9-more-features)
+- [Step 10: Deploying the app to Taipy Cloud](#step-10-deploying-the-app-to-taipy-cloud)
```

Comment:
I would remove it. You already have the table of content displayed on the right panel.

Reply:
Fixed